commands/createfixpullrequests.go

```diff
@@ -195,22 +237,52 @@ func (cfp *CreateFixPullRequestsCmd) openFixingPullRequest(impactedPackage, fixB
 	return cfp.details.Client.CreatePullRequest(context.Background(), cfp.details.RepoOwner, cfp.details.RepoName, fixBranchName, cfp.details.Branch, pullRequestTitle, prBody)
 }
 
-func (cfp *CreateFixPullRequestsCmd) createFixingBranch(impactedPackage string, fixVersionInfo *utils.FixVersionInfo) (fixBranchName string, err error) {
-	fixBranchName, err = cfp.gitManager.GenerateFixBranchName(cfp.details.Branch, impactedPackage, fixVersionInfo.FixVersion)
+// When aggregate mode is active, there can be only one updated pull request to contain all the available fixes.
+// In case of an already opened pull request, Frogbot will only update the branch.
+func (cfp *CreateFixPullRequestsCmd) openAggregatedPullRequest(fixBranchName string, versionsMap map[string]*utils.FixVersionInfo) (err error) {
+	log.Info("Checking if there are changes to commit")
+	isClean, err := cfp.gitManager.IsClean()
+	if err != nil {
+		return
+	}
+	if isClean {
+		return fmt.Errorf("there were no changes in the fix branch '%s'", fixBranchName)
+	}
+	commitMessage := cfp.gitManager.GenerateAggregatedCommitTitle()
+	log.Info("Running git add all and commit...")
+	err = cfp.gitManager.AddAllAndCommit(commitMessage)
+	if err != nil {
+		return
+	}
+	exists, err := cfp.gitManager.BranchExistsInRemote(fixBranchName)
+	if err != nil {
+		return
+	}
+	log.Info("Pushing fix branch:", fixBranchName, "...")
+	err = cfp.gitManager.Push(true)
 	if err != nil {
 		return
 	}
+	if !exists {
+		pullRequestTitle := utils.AggregatedTitleTemplate
+		log.Info("Creating Pull Request form:", fixBranchName, " to:", cfp.details.Branch)
+		prBody := commitMessage + "\n\n" + utils.WhatIsFrogbotMd
```

Comment:
I suggest we leave the PR body empty for now, till we decide what its format should be.

Reply:
The body now is the commit message + link to Frogbot as it was, Do we want to be empty ?